internal/policy/verify.go

```diff
@@ -682,6 +682,15 @@ func verifyTagEntry(ctx context.Context, repo *git.Repository, policy *State, en
 		return err
 	}
 
+	entryTagRef, err := repo.Reference(plumbing.ReferenceName(entry.RefName), false)
```

Comment:
If you set this to `true`, does it resolve to the target?

Reply:
Resolving it will either keep it the same if it is the targets hash, or if it is the tag's hash, it will change it into its target, so at the end it is just better to resolve it. Fixed.